README.md

```diff
@@ -9,17 +9,39 @@ This repository contains a collection of engines that power the core stack for
 Client](https://www.prisma.io/client) and [Prisma
 Migrate](https://www.prisma.io/migrate).
 
-The engines and their respective binary crates are:
+If you're looking for how to install Prisma or any of the engines, the [Getting
+Started](https://www.prisma.io/docs/getting-started) guide might be useful.
 
-- Query engine: `query-engine`
-- Migration engine: `migration-engine-cli`
-- Introspection engine: `introspection-engine`
-- Prisma Format: `prisma-fmt`
+This document describes some of the internals of the engines, and how to build
+and test them.
+
+## What's in this repository
+
+This repository contains four binaries:
```

Comment:
I am pushing that we use "engine" as the term for the thing, that can be used as a binary, (Node-API) library and partially Wasm module. For our public documentation we have pretty much achieved that:
```suggestion
This repository contains four engines:
```